contracts/token/RarePizzasBox.sol

```diff
@@ -110,12 +110,12 @@ contract RarePizzasBox is
         require(msg.value >= price, 'price too low');
         payable(msg.sender).transfer(msg.value - price);
 
-        // presale addresses can purchase up to X total
+        // Presale addresses can purchase up to X total
         _presalePurchaseCount[msg.sender] += 1;
         _purchased_pizza_count.increment();
         _internalMintWithArtwork(msg.sender);
 
-        // BUY ONE GET ONE FREE!
+        // BUY SECOND LAST PIZZA GET LAST ONE FOR FREE!
```

Comment:
let's keep as before.  there's even a wikipedia page for BOGO

https://en.wikipedia.org/wiki/Buy_one,_get_one_free